ftplugin/gohtmltmpl.vim

```diff
@@ -1,8 +1,7 @@
 if exists("b:did_ftplugin")
   finish
 endif
-let b:did_ftplugin = 1
 
-setlocal commentstring=<!--\ %s\ -->
+runtime! ftplugin/html.vim
```

Comment:
Is this available by default for all Vim versions?

Reply:
The `git log` for `runtime/ftplugin/html.vim` goes back to 7.0001, so it seems so